se/sics/mspsim/core/MSP430.java

```diff
@@ -48,11 +49,13 @@ public class MSP430 extends MSP430Core {
   private int[] execCounter;
   private int[] trace;
   private int tracePos;
+  private int prevPc = 0;
```

Comment:
No need to initialize this one, 0 is the default value for fields of numeric types.